app/views/purchases/index.html.erb

```diff
@@ -1,34 +1,75 @@
-<%= render 'header' %>
+<h1 class="title">Purchases</h1>
 
-<div class='purchase-new'>
-  <%= render 'form' %>
+<div class="card">
+  <div class="card-content">
+    <%= render 'form' %>
+  </div>
 </div>
 
+<br>
+
 <div class='purchase-index'>
-  <table>
-    <thead>
-      <tr>
-        <th>Date</th>
-        <th>Description</th>
-        <th>Amount</th>
-        <th>Payee</th>
-        <th>Category</th>
-        <th>Purchaser</th>
-      </tr>
-    </thead>
-    <tbody>
-    <% if @purchases %>
-      <% @purchases.order(:date).reverse_order.each do |purchase| %>
+  <div>
+    <table class="table is-bordered is-striped is-hidden-mobile">
+      <thead>
         <tr>
-          <td><%= purchase.date.to_s %></td>
-          <td><%= purchase.description %></td>
-          <td><%= number_to_currency(purchase.amount) %></td>
-          <td><%= purchase.payee %></td>
-          <td><%= Subcategory.find(purchase[:subcategory_id]).name %></td>
-          <td><%= User.find(purchase[:user_id]).name %></td>
+          <th>Date</th>
+          <th>Description</th>
+          <th>Amount</th>
+          <th>Payee</th>
+          <th>Category</th>
+          <th>Purchaser</th>
         </tr>
+      </thead>
+      <tbody>
+        <% if @purchases %>
+          <% @purchases.order(:date).reverse_order.each do |purchase| %>
```

Comment:
You could consider ordering the purchases in reverse chronological order when you set `@purchases` in your controller. Since probably you'd always want the order to be reverse chronological by default.

Reply:
Good suggestion - thanks!